### DISCUSSION TITLE
Import font from FontShare into TailWind React App

### DISCUSSION THREAD

**Discussion by WilliamLayton** [tailwindlabs/tailwindcss#12603] [Help]
Hi There,
I've seen this around on a few forums, but I'm stuck trying to figure out how to install/import a font from fontshare into my tailwind js config. 

Font Link: https://www.fontshare.com/fonts/general-sans#notice-to-user

The API Link: 

*link href="https://api.fontshare.com/v2/css?f[]=general-sans@500,300,600,400,700&display=swap" rel="stylesheet"*

And then they provide you with this to place in your CSS:

*font-family: 'General Sans', sans-serif;*

How do I extend the themes in my tailwind js config to support these?

Thank you so much in advance.
Will

**Comment by wongjn** [comment]
https://tailwindcss.com/docs/font-family#customizing-your-theme
